libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.spec.ts

```diff
@@ -0,0 +1,320 @@
+        SkyThemeMode.presets.light,
+      ),
+      previousSettings: mockThemeSvc.settingsChange.value.currentSettings,
+    });
+    fixture.detectChanges();
+    await fixture.whenStable();
+
+    expect(fixture.nativeElement.querySelector('sky-icon')).toBeNull();
+    expect(fixture.nativeElement.querySelector('sky-icon-stack')).toExist();
+  });
+
+  it('should set aria label with labelText', async () => {
+    component.labelText = 'test component';
+
+    fixture.detectChanges();
+
+    await checkAriaPropertiesAndAccessibility(
+      'Show help content for test component',
+      null,
+      null,
+    );
+  });
+
+  it('should use aria label with labelText over deprecated ariaLabel input', async () => {
+    component.labelText = 'test component';
+    component.ariaLabel = 'deprecated';
+
+    fixture.detectChanges();
+
+    await checkAriaPropertiesAndAccessibility(
+      'Show help content for test component',
+      null,
+      null,
+    );
+  });
+
+  it('should set help popover when popoverContent input is set', async () => {
+    component.popoverContent = 'content';
+    fixture.detectChanges();
+
+    const { popoverHarness } = await getPopoverTestHarness();
+    await popoverHarness.clickPopoverButton();
+
+    expect(await (await popoverHarness.getPopoverContent()).getBodyText()).toBe(
+      'content',
+    );
+  });
+
+  it('should render help popover title', async () => {
+    component.popoverTitle = 'title';
+    component.popoverContent = 'content';
+    fixture.detectChanges();
+
+    const { popoverHarness } = await getPopoverTestHarness();
+    await popoverHarness.clickPopoverButton();
+
+    expect(await (await popoverHarness.getPopoverContent()).getBodyText()).toBe(
+      'content',
+    );
+    expect(
+      await (await popoverHarness.getPopoverContent()).getTitleText(),
+    ).toBe('title');
+  });
+
+  it('should render help popover if helpContext is a templateRef', async () => {
+    component.popoverContent = component.popoverTemplate;
+    fixture.detectChanges();
+
+    const { popoverHarness } = await getPopoverTestHarness();
+    await popoverHarness.clickPopoverButton();
+
+    expect(await (await popoverHarness.getPopoverContent()).getBodyText()).toBe(
+      'this is a template',
+    );
+  });
+
+  it('should set ariaControls to popover id if popover content is set', fakeAsync(() => {
+    component.popoverContent = 'content';
+    fixture.detectChanges();
+
+    const helpButton = getHelpButton(fixture);
+    helpButton.click();
+    tick();
+    fixture.detectChanges();
+
+    const popoverElementId =
+      debugElement.nativeElement.querySelector('sky-popover').id;
+    const helpInlineElement =
+      fixture.nativeElement.querySelector('.sky-help-inline');
+
+    expect(helpInlineElement?.getAttribute('aria-controls')).toBe(
+      popoverElementId,
+    );
+  }));
+
+  it('should switch the ariaExpanded value with the help inline popover opening and closing', fakeAsync(() => {
```

Comment:
I'm still struggling to understand this sentence. Do you mean, "It should toggle the ariaExpanded value when the popover opens and closes"?

Reply:
that's a much better way to word it 😂